ONSdigital/srm-rh-ui
# SDCSRM 621 Update Contact Details

# Motivation and Context
We've been told the contact details are wrong and so need to be updated

# What has changed
Updated Contact Details
Updated Translations
Updated README section on translation since it was outdated

# How to test?
Build it and check it looks ok and matches what's on the card
Check the Translation works (If you speak Welsh check it's translated correctly)

# Links
[JIRA - SDCSRM-621](https://jira.ons.gov.uk/browse/SDCSRM-621)

# Screenshots (if appropriate):
